modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-power-rpm-{context}"]
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Procedure
+
+. Enter the command:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Enter the command:
++
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. Enter the command:
++
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id> <1>
+----
++
+<1> Pool ID for an active {product-title} subscription
+
+. Enter the command:
++
+[source,terminal]
+----
+# subscription-manager repos --enable="openshift-serverless-1-for-rhel-8-ppc64le-rpms"
+----
+
+. Enter the command:
```

Comment:
Rather than "enter the command", please add more descriptive steps here about what the commands do

Reply:
Yes, when we get time we will make these improvements. We have only two writers for a very large engineering team so this isn't high on our priority list at the moment, but it should be made clearer for new docs like this where possible and we will make improvements to older docs as we get resources to do so.